sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/configuration-reference.md

```diff
@@ -20,16 +20,17 @@ Configuration Reference:
 | `spark.cosmos.useGatewayMode`      | `false`    | Use gateway mode for the client operations  |
 | `spark.cosmos.read.forceEventualConsistency`  | `true`    | Makes the client use Eventual consistency for read operations instead of using the default account level consistency |
 | `spark.cosmos.applicationName`      | None    | Application name  |
-| `spark.cosmos.preferredRegionsList`      | None    | Preferred regions list to be used for a multi region Cosmos DB account. This is a comma separated value (e.g., `[eastus,westus]`) provided preferred regions will be used as hint. You should use a collocated spark cluster with your Cosmos DB account and pass the spark cluster region as preferred region. See list of azure regions [here](https://docs.microsoft.com/dotnet/api/microsoft.azure.documents.locationnames?view=azure-dotnet) |
+| `spark.cosmos.preferredRegionsList`      | None    | Preferred regions list to be used for a multi region Cosmos DB account. This is a comma separated value (e.g., `[eastus,westus]` or `eastus,westus`) provided preferred regions will be used as hint. You should use a collocated spark cluster with your Cosmos DB account and pass the spark cluster region as preferred region. See list of azure regions [here](https://docs.microsoft.com/dotnet/api/microsoft.azure.documents.locationnames?view=azure-dotnet) |
```

Comment:
Looking at https://msdata.visualstudio.com/CosmosDB/_git/CosmosDB?path=%2FProduct%2FMicrosoft.Azure.Documents%2FSharedFiles%2FLocationNames.cs&_a=contents&version=GBmaster I believe the sample values are incorrect? "East US, West US" I think would be the correct format - or do we expect different value sin .Net and Java?

Reply:
all should work, the implementation does both lowercasing and also removes whitespaces. "East US", "EASTUS", "east us", "eastus" all work.

Reply:
done. I changed the doc to "East US, West US".